src/pages/gov/ProposalsByStatus.tsx

```diff
@@ -11,17 +11,18 @@ import { Toggle } from "components/form"
 import ProposalItem from "./ProposalItem"
 import GovernanceParams from "./GovernanceParams"
 import styles from "./ProposalsByStatus.module.scss"
+import { useNetworkName } from "data/wallet"
 
 const ProposalsByStatus = ({ status }: { status: Proposal.Status }) => {
   const { t } = useTranslation()
-  const isClassic = useIsClassic()
+  const networkName = useNetworkName()
 
-  const [showAll, setShowAll] = useState(!isClassic)
+  const { data: whitelist, ...whitelistState } = useTerraAssets<{ [key: string]: number[] }> ("/station/proposals.json")
```

Comment:
` { data, ... ` instead of `{ data: whitelist, ...` and

Reply:
data is already defined below, I renamed it to whitelistData and applied all the other changes